<launch>
  <!-- Control topic to publish to -->
  <arg name="control_topic" default="/control/control"/>
  <!-- Cone topic to subscribe to -->
  <arg name="cone_topic" default="/perception/cone_image_map"/>
  <!-- Use simulated sensors -->
  <arg name="simulating" default="false"/>
  <group ns="control">
    <group ns="flat_image">
      <!-- Launch thresholding node -->
      <node pkg="cone_midpoint_control" type="cone_midpoint_control" name="cone_midpoint_control" output="screen" required="true">
        <param name="control_topic" value="$(arg control_topic)"/>
        <param name="cone_topic" value="$(arg cone_topic)"/>
      </node>
    </group>
  </group>
</launch>
